web/src/components/Selector.tsx

```diff
@@ -92,9 +92,13 @@ const Selector = ({
           <Element
             data-testid={item.code}
             key={item.code}
+            role='option'
             as={Link}
             to={item.href}
             onClick={closeDropDown}
+            onKeyPress={closeDropDown}
+            aria-selected={item.code === activeItemCode}
+            tabIndex={0}
```

Comment:
What happens if this tabIndes is not set? I would think it would work in the same way with the "natural" order

Reply:
i mean maybe it could work depending on browser and the convention is made for older browsers to ensure tab/focus works
But by default `<span>` `<div>` seem not to be focusable/tabable, so the rule forces you in this case for a span which was made "interactive" to set a tabIndex
https://bitsofco.de/how-and-when-to-use-the-tabindex-attribute/

Reply:
Nice read, didn't know about that :+1: